cmd/operator-sdk/bundle/output.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bundle
+
+import (
+	"encoding/json"
+	"log"
+)
+
+// OutPut is an object which is used to return results in the JSON format
+type OutPut struct {
+	IsOK    bool     `json:"isOK"`
+	Message string   `json:"message,omitempty"`
```

Comment:
There should always be a message, even if it is empty.
```suggestion
	Message string   `json:"message"`
```